src/inmanta/ast/statements/define.py

```diff
@@ -479,8 +479,16 @@ def evaluate(self) -> None:
         constraint_type.comment = self.comment
         constraint_type.basetype = basetype
         constraint_type.constraint = self.expression
-        self.expression.normalize()
```

Comment:
This was another statement that was normalized twice. I worked around it using the same approach we use for some other `Define...` statements. Though this should be even less noticeable than the plugins.